storage/dvo_recommendations_storage.go

```diff
@@ -249,16 +287,119 @@ func (storage DVORecommendationsDBStorage) WriteReportForCluster(
 }
 
 func (storage DVORecommendationsDBStorage) updateReport(
-	_ *sql.Tx,
-	_ types.OrgID,
-	_ types.ClusterName,
-	_ types.ClusterReport,
-	_ []types.ReportItem,
-	_ time.Time,
-	_ time.Time,
-	_ time.Time,
+	tx *sql.Tx,
+	orgID types.OrgID,
+	clusterName types.ClusterName,
+	report types.ClusterReport,
+	recommendations []types.WorkloadRecommendation,
+	lastCheckedTime time.Time,
 ) error {
-	// TODO
-	log.Debug().Msg("updating report in database")
+	// Get reported_at if present before deletion
+	reportedAtMap, err := storage.getReportedAtMap(orgID, clusterName)
```

Comment:
I would move this after line 304 (why query the storage if we're going to return nil and do nothing anyway)

Reply:
TY!